storyboard.md

```
@@ -0,0 +1,922 @@
+
+  // Keep refs up-to-date
+  useEffect(() => {
+    principalRef.current = principal;
+    apyRef.current = apy;
+  }, [principal, apy]);
+
+  // Compute per-tick rate as BigInt (amount added per second in smallest unit)
+  // Uses high precision multiplication to avoid rounding down to zero.
+  const perSecondRate = useMemo<bigint>((): bigint => {
+    if (apy <= 0 || principal <= 0n) return 0n;
+    // rate = (principal * apy) / SECONDS_PER_YEAR
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)